components/coreapi/commitment.go

```diff
@@ -0,0 +1,19 @@
+package coreapi
+
+import (
+	iotago "github.com/iotaledger/iota.go/v4"
+)
+
+func getCommitment(index iotago.SlotIndex) (*commitmentInfoResponse, error) {
+	commitment, err := deps.Protocol.MainEngineInstance().Storage.Permanent.Commitments().Load(index)
+	if err != nil {
+		return nil, err
+	}
+
+	return &commitmentInfoResponse{
```

Comment:
use the iota.go serialization of the iotago.Commitment. just call `commitment.Commitment()` to get it

Reply:
not sure if we're going to add more fields here, so we still add a struct here.